### ISSUE TITLE
[Windows] Major RAM overload with flux

### ISSUE THREAD

**Issue by rafalswierczek** [lllyasviel/stable-diffusion-webui-forge#2401]
I have RAM overload during image generation (at the end it spikes drastically).
It starts at 2 GB, then 12 GB and then overload 50+ GB and over 30 GB disc space as fallback.
There is nothing in console - VRAM usage is over 21 GB as expected.

This is on fresh install.

I use T5xxl fp16 text encoder with fp32 Flux 1 dev model.

Below it shows 31 GB but it's over 50 GB in reality (max 64 GB):

![image](https://github.com/user-attachments/assets/f910eb82-83aa-4123-9c20-577fafec6e22)
![image](https://github.com/user-attachments/assets/de8e889a-3559-4c19-8e5f-8e482fac6fde)

**Comment by BenDes21** [comment]
Yes many people experiencing this issue... hope it's will be fixed soon

**Comment by crsongbird** [comment]
I've just started having this issue too!

I run flux in FP8 on a system with 8gb VRAM and 32GB system RAM (which I understand is basically the bear minimum for flux), and it was working WELL with approximately 1 month old code. Yesterday I did an update of my local repo and it completely broke flux too - memory usage skyrocketsm and the system OOMs as expected. It wasn't doing this before.

It seems to be related the hires fix implementation with flux but I'm not completely sure? I know it's behaving a bit differently than it was before and seems to use way more memory during the hires fix than it used to.

This repo [stable-diffusion-webui-ux-forge](https://github.com/anapnoe/stable-diffusion-webui-ux-forge) is approximately 2 months behind this repo and does not have this issue. Since ux-forge is a fork of this repo, it's possible that the issue lies in one of the commits ux-forge does not have.

**Comment by justpretending2** [comment]
Commit `c9850c84362cee0acc4ceb6546f73de59537d167` is good.

**Comment by BenDes21** [comment]
> Commit `c9850c84362cee0acc4ceb6546f73de59537d167` is good.

fixed nothing for me
